addons/mail/static/src/core/web/mention_list.js

```diff
@@ -37,17 +37,14 @@ export class MentionList extends Component {
                         delimiter: this.props.type === "partner" ? "@" : "#",
                         term: this.state.searchTerm,
                     });
-                    const suggestions = this.suggestionService.searchSuggestions(
+                    const { suggestions } = this.suggestionService.searchSuggestions(
                         {
                             delimiter: this.props.type === "partner" ? "@" : "#",
                             term: this.state.searchTerm,
                         },
                         { sort: true }
                     );
-                    this.state.options = [
-                        ...suggestions.mainSuggestions,
-                        ...suggestions.extraSuggestions,
-                    ];
+                    this.state.options = [...suggestions];
```

Comment:
not sure if copy is needed, probably not as I would assume `searchSuggestions` to return a new list every time.

Reply:
indeed, removing spread then